packages/extension-ui/src/components/FaviconBox.tsx

```diff
@@ -56,12 +56,16 @@ export default styled(FaviconBox)(
   border-radius: 8px;
   transition: 0.2s ease;
   justify-content: space-evenly;
+  
+  min-width: 0; /* To enable ellipsis for flex grandchildren (https://css-tricks.com/flexbox-truncated-text/) */
 
   .favicon-box-content {
     display: flex;
     flex-direction: row;
     align-items: center;
     gap: 8px;
+
+    min-width: 0; /* To enable ellipsis for flex grandchildren (https://css-tricks.com/flexbox-truncated-text/) */
```

Comment:
Both levels (= all levels) of descendants of the flex ancestor need `min-width: 0` for it to work.